dashboard/test/ui/features/step_definitions/experiment_steps.rb

```diff
@@ -1,13 +1,3 @@
-When /^I enable the "([^"]*)" course experiment$/ do |experiment_name|
-  steps <<-STEPS
-    Given I am on "http://studio.code.org/experiments/set_course_experiment/#{experiment_name}"
-    And I get redirected to "/" via "dashboard"
-    And I get redirected to "/home" via "dashboard"
-    And I wait to see ".alert-success"
-    Then element ".alert-success" contains text "You have successfully joined the experiment"
-  STEPS
-end
```

Comment:
this ui test step has been unused since https://github.com/code-dot-org/code-dot-org/pull/45489